pkg/kubectl/resource/builder_test.go

```diff
@@ -843,7 +846,7 @@ func TestIgnoreStreamErrors(t *testing.T) {
 	singular := false
 
 	err := b.Do().IntoSingular(&singular).Visit(test.Handle)
-	if err != nil || singular || len(test.Infos) != 2 {
+	if err == nil || singular || len(test.Infos) != 2 {
```

Comment:
@feihujiang, I think we can delete this TestIgnoreStreamErrors test, because there is a TODO in this function saying:
```
order seems bad, but allows clients to determine what they want...
```
Your changes to EagerVistorList.Visit() and StreamVisitor.Visit() fixed this problem. Now we always receive the aggregated error no matter in what order ContinueOnError() and Stream() are called, so this test behaves the same as the next TestReceiveMultipleErrors test.